src/api/interface.ts

```diff
@@ -52,6 +52,8 @@ interface IntentResolution {
   version: string;
 }
 
+type IntentListenerResponse = Promise<{data:any}> | void;
```

Comment:
Can you please add a brief comment to explain this type?

Reply:
done